ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -63,30 +65,30 @@ to answer all of them.
    to be handed off to the responsible Area Director?
 
 10. Several IETF Areas have assembled [lists of common issues that their
-    reviewers encounter][6]. Do any such issues remain that would merit specific
-    attention from subsequent reviews?
+    reviewers encounter][6]. Are any such issues still present in the document?
 
-11. What type of RFC publication is being requested on the IETF stream (Best
-    Current Practice, Proposed Standard, Internet Standard, Informational,
-    Experimental, or Historic)? Why is this the proper type of RFC? Do all
-    Datatracker state attributes correctly reflect this intent?
+11. What type of RFC publication is being requested on the IETF stream ([Best
+    Current Practice][13], [Proposed Standard, Internet Standard][14],
+    [Informational, Experimental or Historic][15])? Why is this the proper type
+    of RFC? Do all Datatracker state attributes correctly reflect this intent?
```

Comment:
Do we really expect the shepherd to recite why it's the appropriate type of RFC? That seems like meaningless busywork.  If someone wants to challenge it, they can explain why the chosen type isn't appropriate.

Reply:
I find the information that is under this question now quite useful, and not an undue burden.

Reply:
The information is indeed useful -- it's the requirement to state a justification which is unreasonable to me. Can we have some (not necessarily inline) examples of actually useful/helpful answers to this question?

Reply:
You'd be surprised how frequently WG beliefs about the status of a document, the text in the document, and the datatracker state disagree.

Reply:
Here are a few examples from recent shepherd write-ups:

```
This document aims to become a Proposed Standard. It does have characteristics
that would also indicate Informational (it is registering code points in FCFS
regions of IANA registries), but we anticipate that this mechanism will be used
and referenced in future normative documents, thus going for PS.
```

```
  The type of RFC being requested is proposed standard. 
  It is expected that future standard track documents will make make use of 
  and reference this document normatively.
  The RFC type is clearly indicated on the title page.
```

```
Internet Standard. This is the proper type of RFC, as CBOR has achieved a high
degree of technical maturity and its implementations have obtained successful
operational experience. This is indicated in the title page header.
```

Not too onerous (certain not compared to the amount of discussion we have had in the WG before IESG submission).

Reply:
Again, I am absolutely unsurprised by disagreements / misalignments about this state, and an effort to assure that they're aligned is well-founded and worth pursuing. On the other hand, also requiring the shepherd to recite why it's the correct status is not -- that's what I'm objecting to. 

@cabo, of the examples you give, the first is a non sequitur (FCFS registration has nothing to do with document status), the second is irrelevant to the requirements for PS, and the third just makes an assertion that mirrors the requirements of the requested status. 

That illustrates my point nicely -- it's not clear what is expected of the shepherd, and thus they're required to either bluster their way through the confusion or copy-and-paste from somewhere else in the hope that it'll be adequate. Vague you-have-to-know-what-they-mean requirements isn't welcoming to newcomers, and doc shepherds are often new people, as it's a good way to get upcoming folks more involved.

A strict reading of 2026 (and the documents that update it) is that:

* Proposed standard 'is stable, has resolved known design choices, has received significant community review, and appears to enjoy enough community interest to be considered valuable.'
* Internet standard additionally requires 'significant implementation and successful operational experience' and 'a high degree of technical maturity'.
* Informational 'is published for the general information of the Internet community, and does not represent an Internet community consensus or recommendation'
* Experimental 'denotes a specification that is part of some research or development effort.'

If paraphrasing one of those statements (or indeed, waving hands about needing to reference things in the future) is adequate to justify the status, why ask for the justification at all, since the requested status has already been stated? 

Otherwise, if there are other requirements or guidelines for choosing a status, where are they written down?

Reply:
Maybe it is intuitively clear to me that this is not about the shepherd inventing a justification, but about reporting, specifically about the WG consideration that has been applied to the question of intended status.
This of course could be clarified.
(The datatracker consistency part of course could be checked automatically.)

Reply:
In that case it'd be much more straightforward to ask something like:

> If there is a non-obvious reason why this status was chosen, please explain why.

It'd also be good if 2026 were updated to fold in the various updates; it's a bit ridiculous to chase them all down.

Reply:
Well, obvious is in the eye of the beholder.  With the security area publishing almost all security standards as informational, very little is obvious to me in this space.

Reply:
I don't disagree -- I think we need more guidance here.

Reply:
Given @cabo's examples, perhaps something like "if not obvious please explain why the current track was chosen."  I don't have the impression that "almost all" security documents are informational; I guess we hang out in different areas within security.

Reply:
"Obvious" isn't the right concept here. Often, it may be obvious to some subset of authors, chairs and responsible ADs, but not for example other ADs. This question is supposed to make such information explicit for the IESG review.

Reply:
The (much) bigger problem is that, to the shepherd (unless they're very experienced in the IETF process) it's more than likely not obvious or even possible to know without asking the "right" people -- that's why I'm pointing this out, and why I think we need more guidance.

Reply:
I'll +1 that handwaving copy/paste reassertments just seem like busywork - or non-transparent gating